taichi/jit/jit_session.cpp

```diff
@@ -33,6 +33,13 @@ std::unique_ptr<JITSession> JITSession::create(TaichiLLVMContext *tlctx,
     return create_llvm_jit_session_cuda(tlctx, config, arch);
 #else
     TI_NOT_IMPLEMENTED
+#endif
+  } else if (arch == Arch::dx12) {
+#ifdef TI_WITH_DX12
```

Comment:
merge with the code block under "#ifdef TI_WITH_LLVM" above?

Reply:
That will result code like return create_llvm_jit_session_cpu(tlctx, config, arch==Arch::dx12?Arch::x64:arch);
Would be nice to keep it separate.